File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/FocusedScepter.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Staff;

/* JADX INFO: loaded from: classes3.dex */
public class FocusedScepter extends Staff {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_staff_focused_scepter_name;
        this.idDescription = R.string.weapon_staff_focused_scepter_description;
        this.idEffect = R.string.weapon_staff_focused_scepter_effect;
        this.idImage = R.drawable.focused_scepter;
        this.price = 854L;
        this.alwaysHits = true;
        this.intelligence = 24;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/FocusedScepter.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Staff

class FocusedScepter : Staff() {
    override fun configureProperties() {
        idName = R.string.weapon_staff_focused_scepter_name
        idDescription = R.string.weapon_staff_focused_scepter_description
        idEffect = R.string.weapon_staff_focused_scepter_effect
        idImage = R.drawable.focused_scepter
        price = 854L
        alwaysHits = true
        intelligence = 24
    }
}
